doc: Project and Org API Key Support Clarification (#1822)
mongodb/terraform-provider-mongodbatlas: website/docs/r/access_list_api_key.html.markdown

# Resource: mongodbatlas_project_ip_access_list_key

`mongodbatlas_access_list_api_key` provides an IP Access List entry resource. The access list grants access from IPs, CIDRs or AWS Security Groups (if VPC Peering is enabled) to clusters within the Project.
    

> -> **Note:** mongodbatlas_project_ip_access_list_key resource can be used to manage all Programmatic API Keys, regardless if they were created at the Organizational level or Project level. 

Reviewer: Code styling and grammar fix:
```suggestion
-> **Note:** The `mongodbatlas_project_ip_access_list_key` resource can be used to manage all Programmatic API Keys, regardless of whether they were created at the Organization level or Project level. 
```
